appveyor.yml

```diff
@@ -0,0 +1,52 @@
+version: 1.0.{build}
+image: Visual Studio 2017
+clone_folder: c:\projects\grisbi-src
+install:
+- cmd: >-
+    c:\msys64\usr\bin\bash -lc "pacman -Syu --noconfirm"
+
+    c:\msys64\usr\bin\bash -lc "pacman -S --noconfirm mingw-w64-x86_64-cairo"
+
+    c:\msys64\usr\bin\bash -lc "pacman -S --noconfirm mingw-w64-x86_64-gtk3"
+
+
+    c:\msys64\usr\bin\bash -lc "pacman -S --noconfirm mingw-w64-x86_64-freetype"
+
+    c:\msys64\usr\bin\bash -lc "pacman -S --noconfirm mingw-w64-x86_64-libgsf"
+    
+    
+    c:\msys64\usr\bin\bash -lc "pacman -S unzip --noconfirm"
+build_script:
+- cmd: >-
+    rem Setting variable is mandatory so that pkg-config to find
+
+    rem all .pc files. Note that we DO NOT want space after MINGW64
+
+
+    set MSYSTEM=MINGW64
+
+    c:\msys64\usr\bin\bash -lc "cd /c/projects/grisbi-src/ && ./autogen.sh"
+
+
+    c:\msys64\usr\bin\bash -lc "cd /c/projects/grisbi-src && ./configure --prefix /c/projects/grisbi-inst/"
+    
+    
+    c:\msys64\usr\bin\bash -lc "cd /c/projects/grisbi-src && v=$(grep PACKAGE_VERSION config.h | cut -f2 -d '\"'); powershell.exe -command \"Update-AppveyorBuild -Version '$v-B%APPVEYOR_BUILD_NUMBER%'\""
+
+
+    c:\msys64\usr\bin\bash -lc "cd /c/projects/grisbi-src && make -j 2"

```

Comment:
You can use "make -j", with no number, to use the maximum of parallelism.
Unless you know that 2 is the best value here.

Reply:
2 is the best value, indeed: https://www.appveyor.com/docs/build-environment/